objcgen/driver.cs

```diff
@@ -36,6 +36,13 @@ public enum TargetLanguage
 		ObjectiveC,
 	}
 
+	public enum BitcodeOption
+	{
+		Default,
+		True,
+		False,
+	}
```

Comment:
like mentioned in https://github.com/mono/Embeddinator-4000/pull/628#discussion_r176771561 a `bool?` gives the same representation and allows the validation code to be reused in other (future) options